proto/panacea/datadeal/v2alpha2/datasale.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package panacea.datadeal.v2alpha2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+import "panacea/oracle/v2alpha2/genesis.proto";
+import "gogoproto/gogo.proto";
+
+// DataSale defines a data sale.
+message DataSale {
+  string seller_address = 1;
+  uint64 deal_id = 2;
+  bytes seller_cid = 3;
+  bytes buyer_cid = 4;
```

Comment:
```suggestion
  bytes oracle_cid = 3;
  bytes buyer_cid = 4;
```
How about naming these as `<receiver>_cid`?

Reply:
Does the `oracle_cid` means the new CID after verifying the seller's data?

Reply:
Without this comment, I fixed the other comments in [7b4eb64](https://github.com/medibloc/panacea-core/pull/360/commits/7b4eb6470c24d5da4f7cb898149d534a698a719e).

Reply:
As I understand, the `seller_cid` indicates a data that is encrypted for oracles so that oracles can validate the data.
And, the `buyer_cid` indicates a data that is encrypted for a buyer.

Reply:
As discussed in Slack, let's go with `verifiable_cid / delivered_cid`.

Reply:
Renamed in [43f8bb2](https://github.com/medibloc/panacea-core/pull/360/commits/43f8bb2b9e8e91ac9f2d94830acba348b2b8f6e4).